fern-logger/src/config.rs

```diff
@@ -227,6 +227,12 @@ pub struct LoggerConfig {
     pub(crate) outputs: Vec<LoggerOutputConfig>,
 }
 
+impl Default for LoggerConfig {
```

Comment:
This will probably need to be released if you cant to release the trace tools crate